components/equihash/src/tromp.rs

```diff
@@ -70,13 +71,21 @@ unsafe fn worker(eq: *mut CEqui, p: verify::Params, curr_state: &State) -> Vec<V
         let nsols = equi_nsols(eq);
         let sols = equi_sols(eq);
         let solution_len = 1 << p.k;
+        //println!("{nsols} solutions of length {solution_len} at {sols:?}");
```

Comment:
I'm inclined to remove this and the other introduced debugging print statements, unless anyone else disagrees.

Reply:
I think we should keep the debug print statements and consider adding more.